hazelcast-spring/pom.xml

```diff
@@ -30,7 +30,7 @@
     </parent>
 
     <properties>
-        <org.springframework.version>3.1.0.RELEASE</org.springframework.version>
+        <org.springframework.version>4.0.2.RELEASE</org.springframework.version>
```

Comment:
I have just tested it. It works with Spring 3.2.0 and JCache 0.5. We should mention in documentation that those who want to use this feature should depend on JCache 0.5.